mmv1/templates/terraform/examples/certificate_manager_certificate_map_entry_basic.tf.erb

```diff
@@ -0,0 +1,10 @@
+resource "google_certificate_manager_certificate_map_entry" "<%= ctx[:primary_resource_id] %>" {
+  name        = "<%= ctx[:vars]['cert_map_entry_name'] %>"
+  description = "My acceptance test certificate map entry"
+   labels      = {
+    "terraform" : true,
+    "acc-test"  : true,
+ }
+#   certificates = []
```

Comment:
Lets copy the code [here](https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/templates/terraform/examples/certificate_manager_certificate_basic.tf.erb) and add at least one certificate in here? I think the basic use case is covered by the `certificate_map` basic test, so we can rename this to be `certificate_manager_certificate_map_entry_full.tf.erb` if we want? to indicate we're testing all fields.